src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -73,8 +62,11 @@ export const getters = {
                     valueColumn = "current_art";
                     break;
                 case (DataType.ANC):
+                    //TODO: once using metadata, we need to allow multiple values per row, as in the case on ANC data
+                    //which contains both prevalence and art data in each row (wide format), unlike survey, which
+                    //is int long format, with an indicator column to show which indicator the value each row provides
                     indicator = "prev";
-                    valueColumn = "ancrt_test_pos";
+                    valueColumn = "prevalence";
```

Comment:
Using the column for prevalence which is actually in the metadata (though not using metadata to identify these columns yet..)